### ISSUE TITLE
question of model(**inputs)

### ISSUE THREAD

**Issue by shiyunchengAI** [yanzhh/HGERE#7]
Hello, I have a question for you.
When model(**input) is used, the model calls the forward function. After I modified the input content, the training error was reported, but I did not find the forward step by step, because I thought it was necessary to modify the corresponding forward function. So I need to ask for your help.

        for step, batch in enumerate(epoch_iterator):
            model.train()
            batch = tuple(t.to(args.device) for t in batch)

            inputs = {'input_ids':      batch[0],
                      'attention_mask': batch[1],
                      'position_ids':   batch[2],
                      'labels':         batch[3],
                      'X':         batch[4],
                      }

            if args.model_type.find('span')!=-1:
                inputs['mention_pos'] = batch[5]
            if args.use_full_layer!=-1:
                inputs['full_attention_mask']= batch[6]

            outputs = model(**inputs)
